### DISCUSSION TITLE
More skyline photos

### DISCUSSION THREAD

**Discussion by syvasha** [endless-sky/endless-sky#8589] [Ideas]
Hello,

I have a number of skyline photos that I have been meaning to share for use in ES for quite some time.
What would be the procedure? A pull request seems overkill, I could upload them somewhere (an album) and let you guys look if you like any, before having to properly cut them etc.

**Comment by syvasha** [comment]
duplicate of
https://github.com/endless-sky/endless-sky/discussions/8573

in a way

**Comment by bene-dictator** [comment]
You could show them on thhe Endless Sky community discord and we could see where they might go, but if you can't do that you should upload an album.
